pkg/lib/dialogs.tsx

```diff
@@ -82,64 +82,69 @@
  *   }
  * }
  *
- * If there is a situation where you want to wait until a Dialog is closed
- * after opening, you can "await Dialogs.show(<My Dialog />)"
+*
+ * - Dialogs.show(component)
```

Comment:
We want to consider show() deprecated now, right? I think it'd make sense to reorganize the documentation to have the .run() documentation and example first, and then the old one under a "deprecated API:" warning.

Reply:
I don't consider `.show()` or `.close()` deprecated.  I prefer the new API, but others may like the old one, and there's really nothing wrong with it (other than the somewhat theoretical potential for one dialog to close another by accident).

We could try to port everything over to the `.run()` API, but I don't see a huge advantage in it for people who aren't trying to return data.

Reply:
OK, then I misunderstood how far you wanted to go go with this.